gradle.properties

```diff
@@ -14,7 +14,8 @@
 # limitations under the License.
 #
 
-
+org.gradle.java.installations.auto-download=false
+org.gradle.java.installations.fromEnv=JDK8,JDK11,JDK16,JDK17
 
```

Comment:
Related to the above - as JDKx is set by you, I wonder, if:
> org.gradle.java.installations.auto-detect=false

couldn't be added to the configuration?

I see that in the 3.0, 11 build on Ubuntu, JDK 11 is "Detected by: Common Linux Locations", but probably it just has a precedence, as the JDK11 variable also seems to be set.
https://github.com/spockframework/spock/pull/1366/checks?check_run_id=3748208999#step:7:55

Reply:
This would force everyone to define those env variables on their workstations as well.

Reply:
In that case, `org.gradle.java.installations.auto-detect=false` in general could be problematic. We could add it only on CI (via `echo "..." >> gradle.properties`), but assuming the locally pre-installed (and auto-detected) version is ok for us (not that one pointed out with the env) there should be no problem and the gain of avoiding auto-detection would be probably negligible.